### DISCUSSION TITLE
I bought a license and I can't use it

### DISCUSSION THREAD

**Discussion by Othniel-am** [aclap-dev/video-downloadhelper#1423] [Help and Q&A]
### Your question

I purchased a license using a debit card and I did not receive a confirmation email nor the license, but the debit was made. When I enter my email on the recover license page the message says that there is no license linked to my email.
purchase date; 31/10/24 às 15:41 (GMT - 03:00) - US$ 18,00

### Addon & CoApp version and details

irrelevant

**Comment by jcfrog** [comment]
Hi,
Please send us an email at licenses@downloadhelper.net
